src/test/perl/path.t

```diff
@@ -108,9 +108,9 @@ sub verify_exception
     } else {
         ok(! $ec_check->error(), "Error reset after $msg");
     };
-    if ($noreset) {
+    if ($noreset && $mc->{fail}) {
```

Comment:
I understand that my problem was coming from the fact I was not using `_safe_eval()`. Nevertheless, I think we should protect in some way against an `undef` argument rather than adding a warning to the error...

BTW, I think there is a typo at the end of `_safe_eval()`. Should be `chomp $res_text` and `verbose()` should print `$res_text`, shouldn't it?